manifests/e/Envoy49/go-spotify-cli/1.0.41/Envoy49.go-spotify-cli.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Envoy49.go-spotify-cli
+PackageVersion: 1.0.41
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent-progress
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Envoy49/go-spotify-cli/releases/download/v1.0.41/go-spotify-cli.exe
+  InstallerSha256: 318B66CC3D0B971B6B5B740E5A744D00B24746366AA057D2F72F2D492C41BF06

```

Comment:
```suggestion
InstallerType: portable
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/Envoy49/go-spotify-cli/releases/download/v1.0.41/go-spotify-cli.exe
  InstallerSha256: 48B5B081BE1F27C422CD46B37DF50A2145A9399BD0A285D63D0E97D86D51C216
```

Reply:
thanks a lot, I applied your suggestion in another PR, hopefully it will go through the build this time

Reply:
@Trenly In one of my PRs I tried that options and it also failed. https://github.com/microsoft/winget-pkgs/pull/132384/files

Reply:
but the architecture type was x64, lets see what will happen this time

Reply:
> @Trenly In one of my PRs I tried that options and it also failed. [#132384 (files)](https://github.com/microsoft/winget-pkgs/pull/132384/files)

I'm not seeing a failure in #132384, I'm only seeing that the PR was created; There aren't any comments indicating a check was even run on the PR.  Perhaps you could try applying the suggestion from above just to see what happens? Even if it fails it will still be a good way to debug between the other PR you have open (#132469) and this one

Reply:
Good point, lets do that. Thanks a lot!!!